openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java

```diff
@@ -115,8 +139,12 @@ public OpenshiftStartedEnvironment(
             boolean tempBuild,
             Date temporaryBuildExpireDate) {
 
+        ReadEnvProperty reader = new ReadEnvProperty();
+        CREATION_POD_RETRY = reader.getIntValueFromPropertyOrDefault(CREATION_POD_RETRY_PROPERTY, DEFAULT_CREATION_POD_RETRY);
```

Comment:
This approach doesn't follow the standard way, how we configure orchestrator - using config file. We should avoid mixing the approaches and follow one.

Reply:
Ok will use config file